.travis.yml

```diff
@@ -1,8 +1,8 @@
 language: dart
 
 dart:
-  - 2.0.0
-  - dev
+  - 2.1.0
+  - stable 
```

Comment:
This should have stayed `dev` – keeps us honest about upcoming SDKs!

Reply:
..... should I send a follow-up PR? It's only been up for a few min!

Reply:
Is there a different between dartvm 2.1.0 stable and dartvm 2.1.0 dev?